homeassistant/components/switch/dlink.py

```diff
@@ -74,37 +78,28 @@ def device_state_attributes(self):
         try:
             ui_temp = self.units.temperature(int(self.data.temperature),
                                              TEMP_CELSIUS)
-            temperature = "%i %s" % \
-                          (ui_temp, self.units.temperature_unit)
+            temperature = ui_temp
```

Comment:
I originally added the temperature_unit to the `temperature` variable to make sure it is written correctly when clicking on the sensor is HASS. Otherwise, HASS will just write the number (ex. 22) but not the unit. Will this change remove the unit from the UI or has HASS changed in some way?

Reply:
Yes, this will remove the unit from the attribute. I think that is standard for other components in HA, and will allow to make automation based on the attributes.

Reply:
As a trained physicist I am not really happy about losing the unit. A number does not really mean much without a unit 😃 
but if it is standard in HASS then let's just follow the standard.

Reply:
I am a mathematician so I do not care about units 😬 :grin: